src/TestStack.Seleno/Configuration/BrowserFactory.cs

```diff
@@ -142,7 +143,8 @@ public static InternetExplorerDriver InternetExplorer()
         public static InternetExplorerDriver InternetExplorer(InternetExplorerOptions options)
         {
             return new WebDriverBuilder<InternetExplorerDriver>(() => new InternetExplorerDriver(options))
-                .WithFileName("IEDriverServer.exe");
+                .WithFileName("IEDriverServer.exe")
+                .WithProcessNames("IEDriverServer", "iexplore");
```

Comment:
Probably also need to do this for chrome, but for now just adding for IE since it was easy for me to test at the time